### ISSUE TITLE
WCAG testing

### ISSUE THREAD

**Issue by SidraShahid0510** [SidraShahid0510/Game-Hub#14]
- [ ] The site should be WCAG compliant. 
- [ ] Use the WAVE Web Accessibility Evaluation Tools to test that site matches best practices for accessibility.
